samples/drivers/led/pwm/src/main.c

```diff
@@ -66,28 +63,38 @@ static void run_led_test(const struct device *led_pwm, uint8_t led)
 			LOG_ERR("err=%d brightness=%d\n", err, level);
 			return;
 		}
-		k_sleep(FADE_DELAY);
+		k_sleep(K_MSEC(CONFIG_FADE_DELAY));
 	}
 	k_sleep(K_MSEC(1000));
 
-	/* Set LED blinking (on: 0.1 sec, off: 0.1 sec) */
-	err = led_blink(led_pwm, led, 100, 100);
+#if CONFIG_BLINK_DURATION_SHORT > 0
+	/* Start LED blinking (short cycle) */
+	err = led_blink(led_pwm, led, CONFIG_BLINK_DURATION_SHORT, CONFIG_BLINK_DURATION_SHORT);
```

Comment:
Optionally, limit the line length to 80 columns, to stay consistent with the rest of the code.